src/pages/ValidateLoginPage.js

```diff
@@ -17,10 +17,11 @@ const defaultProps = {
 };
 class ValidateLoginPage extends Component {
     componentDidMount() {
+        // TODO: figure out if accountID is needed anymore
         const accountID = lodashGet(this.props.route.params, 'accountID', '');
```

Comment:
Still figuring out if we want to pass this `accountID` to `Session.signIn` so the PHP layer can figure out the user's email

Reply:
Convo here: https://github.com/Expensify/Web-Expensify/pull/35881/files#r1064381238

Reply:
Ok turns out I'm not going to edit this file at all, that'll be modified in a different issue